app/templates/todo-list.hbs

```diff
@@ -0,0 +1,56 @@
+{{page-title "TodoList"}}
+<div class="pad20">
+    <div class="todo-bar">
+        <div>
+            Todo List
+        </div>
+        
+        <div>
+            <Button><LinkTo @route="add-todo">+ Add Todo</LinkTo></Button>
```

Comment:
`<LinkTo>` renders an anchor tag `a`. We should not render an `a` tag inside a `button` tag.

Reply:
If you want it as a button, style the link with CSS